src/components/HomePage.jsx

```diff
@@ -0,0 +1,5 @@
+const Homepage = () => {
```

Comment:
I don't believe you need this component anymore as the ArticlesList is acting as your homepage? Can probably get rid 🙂